### PR TITLE
Fixes issue #711

### PR DESCRIPTION
**Pull request by Gillardo**
Popup.pollPopup immediately closes the popup if the authorization server and the JS app share the same address.  This fixes the problem

### CODE REVIEW

**Comment by Gillardo** [comment]
Resubmitted PR, as code wasnt working when registering client in asp.net.  Now all fixed, as was checking redirect_uri incorrectly on the first PR.  This PR works on all cases

File: satellizer.js [hunk 1]
```diff
@@ -742,15 +742,16 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' && module.ex
...
+        Popup.pollPopup = function(redirectUri) {
           var deferred = $q.defer();
 
           var polling = $interval(function() {
             try {
-              var documentOrigin = document.location.host;
-              var popupWindowOrigin = Popup.popupWindow.location.host;
-
-              if (popupWindowOrigin === documentOrigin && (Popup.popupWindow.location.search || Popup.popupWindow.location.hash)) {
+			  var popupWindowOrigin = Popup.popupWindow.location.protocol + '//' +
```
Comment (PinpointTownes) [inline satellizer.js]: I see you've reused my quick and dirty fix as-is :smile: 

That would be great if you could test it with well-known providers (e.g Google, Facebook) to make sure it behaves properly.

**Comment by Gillardo** [comment]
Its was a better change than mine. I didnt realose that the redirectUrl was available in the calling function.  I have tested this with google and facebook. I have also tested it when the user doesnt exist and has to register using an asp.net vnext website. All work perfectly

**Comment by sahat** [comment]
Thank you @Gillardo and @PinpointTownes. I will do some more testing and create a new version tag shortly.

**Comment by PinpointTownes** [comment]
@sahat thanks! :clap:

**Comment by stefanmaric** [comment]
I was just trying the example code using master#a6706b05ba72970a189badee2a237b85f30417da and LinkedIn provider. It is not working for me. Seems `pollPopup` is never resolved.

Sorry the little detail I can provide.

**Comment by viniciusdacal** [comment]
@stefanmaric same problem here.

**Pull request merged**